hugr-py/src/hugr/serialization/tys.py

```diff
@@ -381,18 +381,17 @@ def deserialize(self) -> tys.PolyFuncType:
 
 
 class TypeBound(Enum):
-    Eq = "E"
     Copyable = "C"
     Any = "A"
 
     @staticmethod
     def join(*bs: TypeBound) -> TypeBound:
```

Comment:
Not covered by tests?

Reply:
probably pulled in from guppy, new issue : #1365